src/App.css

```diff
@@ -20,3 +20,16 @@
 .App-link {
   color: #09d3ac;
 }
+
+.Header-app-bar {
```

Comment:
For the header, can you make a folder just for header and put the css and component in there?